resources/migrations/049_add_pulse_skip_if_empty.yaml

```diff
@@ -0,0 +1,15 @@
+databaseChangeLog:
+  - changeSet:
+      id: 49
+      author: tlrobinson
+      changes:
+        ######################################## add skip_if_empty column to pulse ########################################
+        - addColumn:
+            tableName: pulse
+            columns:
+              - column:
+                  name: skip_if_empty
+                  type: boolean
```

Comment:
it would also be nice if we could have `remarks` about the purpose of this column for people hacking on Metabase (see some of the more recent migrations for examples of how to do that)